src/ast/types/blocks-and-statements.ts

```diff
@@ -1,39 +1,86 @@
 import { Identifier, UnannType } from "./classes";
 
-export type BlockStatement = LocalVariableDeclarationStatement;
+export interface Block {
+  kind: "Block";
+  blockStatements: Array<BlockStatement>;
+}
+export type BlockStatement = LocalVariableDeclarationStatement | Statement;
+
 export interface LocalVariableDeclarationStatement {
   kind: "LocalVariableDeclarationStatement";
   localVariableType: LocalVariableType;
-  variableDeclarationList: VariableDeclarator;
+  variableDeclaratorList: Array<VariableDeclarator>;
+}
+
+export type Statement = StatementWithoutTrailingSubstatement | IfStatement | WhileStatement | DoStatement | ForStatement;
```

Comment:
Is `DoStatement` not under `StatementWithoutTrailingSubstatement`?

Reply:
my bad, it should be under `StatementWithoutTrailingSubstatement`, changed